src/main/java/io/appium/java_client/AppiumCommandInfo.java

```diff
@@ -16,12 +16,14 @@
 
 package io.appium.java_client;
 
+import lombok.AccessLevel;
+import lombok.Getter;
 import org.openqa.selenium.remote.CommandInfo;
 import org.openqa.selenium.remote.http.HttpMethod;
 
 public class AppiumCommandInfo extends CommandInfo {
-    private final String url;
-    private final HttpMethod method;
+    @Getter(AccessLevel.PUBLIC) private final String url;
```

Comment:
`Public` need not to be set explicitly in Lombok AFAIK. But no harm

Reply:
Since prior implementation was public method call, i have implemented the same.